### ISSUE TITLE
Accessibility indicator on greeter

### ISSUE THREAD

**Issue by cassidyjames** [elementary/greeter#315]
We should be able to access an accessibility indicator to launch the screen reader on the greeter.

**Comment by worldofpeace** [comment]
Guess that'd be https://github.com/elementary/wingpanel-indicator-a11y

**Comment by danrabbit** [comment]
This has been fixed!
